### DISCUSSION TITLE
License of dependency package (@mikaelkristiansson/domready) is listed as Unlicensed

### DISCUSSION THREAD

**Discussion by Gravedigger7789** [gatsbyjs/gatsby#29374] [Help]
I am running https://github.com/microsoft/license-checker-webpack-plugin and it is outputting that a dependency of gatsby is unlicensed
`License: @mikaelkristiansson\domready@1.0.10 is unlicensed`

After visiting the npm url https://www.npmjs.com/package/@mikaelkristiansson/domready it is also listed with no license there as well.

It seems like the author may have missed adding the license line to the package.json file.

I just wanted to double check that this dependency is allowed to be used.

**Comment by vladar** [comment]
This package has a [license on GitHub](https://github.com/mikaelkristiansson/domready/blob/master/LICENSE) and in [package.json](https://github.com/mikaelkristiansson/domready/blob/b5ca782ddf5663b28f12935190a89bd4a1796d9b/package.json#L7) But looks like this was not published in npm. I've asked the question about it [here](https://github.com/mikaelkristiansson/domready/pull/1). Hopefully, they will publish it.

I am not a lawyer but I assume a license on GitHub should be enough to allow its usage.
